src/ansys/dpf/core/plotter.py

```diff
@@ -87,6 +87,39 @@ def add_scale_factor_legend(self, scale_factor, **kwargs):
             **kwargs_in,
         )
 
+    def add_points(self, points, field):
+        import pyvista as pv
+
+        point_cloud = pv.PolyData(points)
+        if field:
+            point_cloud[f"{field.name}"] = field.data
+        self._plotter.add_points(point_cloud)
+
+    def add_line(self, points, field=None):
+        import pyvista as pv
+
+        line_field = pv.PolyData(np.array(points))
+        if field:
+            line_field[f"{field.name}"] = field.data
+            self._plotter.add_mesh(line_field)
+        else:
+            self._plotter.add_lines(points)
+
+    def add_plane(self, center, direction, field=None):
+        import pyvista as pv
+
+        plane = pv.Plane(
+            center=center,
+            direction=direction,
+            i_size=0.005,
+            j_size=0.005,
+            i_resolution=20,
+            j_resolution=20,
+        )
```

Comment:
Shouldn't we use here the discretization?

Reply:
The discretized objects are just to interpolate there later. Conceptually, I think a user should be able to see t'he plane even if It has nit been discretized.

Having said that, I was planning on changing this and discretize all objects right after discretizing. The reason being is that, for the planes, it makes a lot of sense asking to users to provide width and height of the plane when they try to create it using a center and a normal. However, if they define 3 points I think It would be more intuitive to consider these 3 points as the corners of the plane, thus, width and height being already defined.

What do you think?? Of course I will allow users to change the width and height of the plane anyway.

If I go with this approach, I Will be able to always plot the plane with its defined width and height and I will modify these lines.

Reply:
> discretize all objects right after discretizing 

I'm not sure I follow.

> asking to users to provide width and height of the plane when they try to create it using a center and a normal. However, if they define 3 points I think It would be more intuitive to consider these 3 points as the corners of the plane, thus, width and height being already defined.

This is not very clear to me. What do you mean with `if they define 3 points I think It would be more intuitive to consider these 3 points as the corners of the plane, thus, width and height being already defined.` ?

Reply:
What I have implemented now is that the objects are discretized right after initialization, so the user will not need to first create a plane and then discretize it. It will be done all in one line.

Regarding the plane. When creating a plane with its center and normal, it makes sense to ask the user to provide width and height of the line as:
```py
plane = create_plane_from_center_and_normal(center=[0,0,0], normal=[0,1,0], width=1, height=1)
```

However, I think that when defining a plane with three points:
```py
plane = create_plane_from_points([[0,0,0], [0,2,0], [0, 0, 4]])
```
the user may expect these three points to be the corner points of the plane. Thus, the width and height will be computed accordingly. For instance, the previous case will return a plane that has width = 2 and height = 4.